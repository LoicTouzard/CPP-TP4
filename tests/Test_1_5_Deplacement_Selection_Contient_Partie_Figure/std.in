C c 0 0 5
S s1 -3 3 3 -3
MOVE s1 2 2
LIST
EXIT
