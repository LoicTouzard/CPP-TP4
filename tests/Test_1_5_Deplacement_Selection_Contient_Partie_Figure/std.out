OK
#New object :c
OK
#New object : s1
OK
C c 0 0 5
